src/football/worker/core/GameSim/index.js

```diff
@@ -132,19 +132,27 @@ class GameSim {
     }
 
     homeCourtAdvantage() {
+        var homeCourtModifier = 1.0 + g.homeCourtAdvantage / 100.0;
```

Comment:
Just FYI, if you're coming from Python, JS has no distinction between ints and floats so you don't need to add .0 to numbers :) no difference either way in JS.